TipCatDotNet.Api/Services/HospitalityFacilities/MemberService.cs

```diff
@@ -250,6 +220,17 @@ async Task<Result> UpdateMember()
         }
 
 
+        private Result Validate(MemberContext memberContext, MemberRequest request, MemberValidateMethods methodType)
+        {
+            var validator = new MemberRequestValidator(memberContext, _context, methodType);
+            var validationResult = validator.Validate(request);
```

Comment:
Suggestion: instead of `validator.Validate` use `validator.ValidateAdd`, which invokes base method within itself. That helps avoid passing `MemberValidateMethods`